PowerShell/JumpCloud Module/Public/Commands/Get-JCCommand.ps1

```diff
@@ -1,10 +1,34 @@
 
 Function Get-JCCommand ()
 {
-    [CmdletBinding(DefaultParameterSetName = 'ReturnAll')]
+    [CmdletBinding(DefaultParameterSetName = 'SearchFilter')]
 
     param
     (
+
+    #### NEW PARAMS
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'The command to execute on the server.')]
+        [String]$command,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'Name of the command')]
```

Comment:
Same as above but I think we can remove the example:

"The name of the JumpCloud Command you wish to search for."